assets/js/components/coinify/sell-summary.component.js

```diff
@@ -16,8 +16,6 @@ angular
   });
 
 function CoinifySellSummaryController ($scope, $q, Wallet, currency, Alerts, $timeout, Exchange, coinify) {
-  this.sellRateForm;
```

Comment:
Im a bit worried about this, did you test that the form still works as intended? @prwelber do you remember the point of this sellRateForm?

Reply:
That should have no affect on the state of the ctrl since its an unused expression (not a declaration or assignment).  However, we had numerous other `no-unused-expresssions` violations that I didn't feel as comfortable changing. Therefore I just put this line back in.